code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -199,6 +199,9 @@
 			FindTarget()
 
 /mob/living/simple_animal/hostile/proc/AttackingTarget()
+	if(target == src)
+		to_chat(src,"Why attack yourself?")

```

Comment:
It really should not be a message. It's inconsistent with other feedback messages used when attacking others, and also sounds and feels very unprofessional. It's what children say to eachother: "Why are you attacking yourself?" "Why are you attacking yourself?". 

I still don't think it's entirely necessary, but it really should not be a question.